ShortestPath/ShortestPath/Network.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ShortestPath
+{
+    public class Network
```

Comment:
Please rename the file to match the class name

Reply:
could you explain a little more?